### PR TITLE
Implemented top method in OpenShiftConnector

### PR DESCRIPTION
**Pull request by JPinkney**
Signed-off-by: jpinkney <josh.pinkney@mail.utoronto.ca>

### What does this PR do?
Rebases https://github.com/eclipse/che/pull/4466 onto master branch.

### What issues does this PR fix or reference?
Rebases https://github.com/eclipse/che/pull/4466
Part of: https://github.com/redhat-developer/rh-che/issues/322 

#### Changelog

Implemented top method in OpenShiftConnector

#### Release Notes

Top method implemented in OpenShiftConnector

### CODE REVIEW

**Comment by benoitf** [comment]
ci-build

**Review by l0rd — APPROVED** [review]

**Comment by benoitf** [comment]
ci-build

File: plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java [hunk 1]
```diff
@@ -1031,6 +1036,72 @@ public void onClose(KubernetesClientException cause) {
...
+    OpenShiftClient openShiftClient = new DefaultOpenShiftClient();
+    try (ExecWatch watch =
+        openShiftClient
+            .pods()
+            .inNamespace(openShiftCheProjectName)
+            .withName(podName)
+            .redirectingOutput()
+            .redirectingError()
+            .exec(command)) {
+      Thread.sleep(2500);
```
Comment (benoitf) [inline plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java]: is it mandatory this wait ?

Comment (JPinkney) [reply plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java]: I asked @snjeza and was told that it was copied from the exec method. I believe it was copied from this https://github.com/JPinkney/che/blob/0c86b9b4c25e045e657a1dbca850fdb1c3c78e5d/plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java#L1136-L1155 as far as I can see the ExecutorService declared isn't even used except for shutting down in the finally block so this line wouldn't be needed.

**Comment by benoitf** [comment]
ci-build

**Review by l0rd — APPROVED** [review]
LGTM

**Pull request merged**